package.json

```diff
@@ -26,7 +26,7 @@
     "start": "electron .",
     "pack-mac": "electron-packager . 'Image Shrinker' --extendInfo=extend.plist --platform=darwin --arch=x64 --ignore=/^release/i --icon=assets/icons/mac/icon_image-shrinker.svg.icns --overwrite --prune=true --out=release-builds",
     "pack-linux": "electron-packager . 'Image Shrinker' --platform=linux --arch=x64 --ignore=/^release/i --icon=assets/icons/png/512x512 --overwrite",
-     "pack-win": "npm install&&electron-packager . Image-Shrinker --asar=true --platform=win32 --ignore=/^release/i --overwrite --icon=assets/icons/ico/512x512@2x.ico --version 1.3.7.0",
+     "pack-win": "npm install&&electron-packager . Image-Shrinker --asar=true --platform=win32 --ignore=/^release/i --overwrite --icon=assets/icons/ico/512x512@2x.ico --app-version 1.3.7.0",
```

Comment:
I think, we should remove npm install here. I want to leave the possibility using different package managers

Reply:
kk

Reply:
<https://github.com/derpmarine168/image-shrinker/blob/patch-1/menu/mainmenu.js#L50> is needed to stop a non-standard menu appearing on win32.